Remotion/Development/UnitTesting.Compilation/UnitTesting/AssemblyCompilationException.cs

```diff
@@ -17,7 +17,7 @@
 using System;
 using System.Runtime.Serialization;
 
-namespace Remotion.Development.UnitTesting
+namespace Remotion.Development.UnitTesting.Compilation.UnitTesting
```

Comment:
We should move this file and the one besides it out one level. We no longer need the `UnitTesting` part as suffix since we already have it before